graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/DefaultDgsQueryExecutor.kt

```diff
@@ -52,12 +53,12 @@ class DefaultDgsQueryExecutor(defaultSchema: GraphQLSchema,
 ) : DgsQueryExecutor {
 
     private val parseContext: ParseContext =
-            JsonPath.using(Configuration.defaultConfiguration()
+            JsonPath.using(Configuration.builder()
```

Comment:
Ahhh - I see. We will want to make this same change in the [client](https://github.com/Netflix/dgs-framework/blob/master/graphql-dgs-client/src/main/kotlin/com/netflix/graphql/dgs/client/GraphQLResponse.kt#L43)

Reply:
Yes, good call, I forgot about that one.

I've pushed the change to client as well.

Reply:
I don't think we really need to support changing out the json provider (like your example in the autoconfig). It would require us to test that scenario as well, which I don't think we can do consistently. 

Unless this comes up as an issue for folks I suggest we just hardcode it to Jackson.

Reply:
> Unless this comes up as an issue for folks I suggest we just hardcode it to Jackson.

I 110% agree and was just sharing the "how I got to the final PR" w/ you.

Reply:
Changes LGTM.